src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -55,32 +55,59 @@ class SearchControls extends Component {
   }
 
   renderServiceFilterOptions() {
-    const { currentQuery } = this.props;
-
-    if (currentQuery.facilityType === 'va_health_facility') {
-      return [
-        <option key="primary_care" value="primary_care">Primary Care</option>,
-        <option key="mental_health" value="mental_health">Mental Health</option>,
-        <option key="more_services" value="more_services" disabled>More services coming soon</option>,
-      ];
+    const { currentQuery: { facilityType } } = this.props;
+
+    switch (facilityType) {
+      case 'health':
+        return [
+          <option key="primary_care" value="primary_care">Primary Care</option>,
```

Comment:
@crwallace this option is overridden by the service filters being disabled for everything except for Benefits but I will remove it on the next PR so that we don't hold up this merge